platform/android/build.gradle.template

```diff
@@ -35,6 +35,9 @@ android {
 	packagingOptions {
 		exclude 'META-INF/LICENSE'
 		exclude 'META-INF/NOTICE'
+		exclude 'res/drawable/icon.png'
+		exclude 'res/layout/downloading_expansion.xml'
+		exclude 'res/layout/status_bar_ongoing_event_progress_bar.xml'
```

Comment:
Update classpath build tools to 2.1.2 to match the latest version of Android Studio as of this commit.

![buildtools_212](https://cloud.githubusercontent.com/assets/12244526/17388744/534386dc-5a28-11e6-91e1-8b5d69335322.png)

:-)